docs/travis.md

```diff
@@ -0,0 +1,16 @@
+# How we use Travis CI
+
+## What Travis does
+We use [Travis CI](https://travis-ci.org/) on cfgov-refresh to perform the following tasks:
+
+- Run automated unit tests, accessibility tests, and acceptance tests.
```

Comment:
I actually should check if this is true

Reply:
I don't know of any particular accessibility tests. We do run unit and acceptance tests.